src/templates/nav-primary.html

```diff
@@ -0,0 +1,31 @@
+<button class="usa-nav-close">
+  <img src="{{ uswds.path }}/img/close.svg" alt="close">
+</button>
+<ul class="usa-nav-primary usa-accordion">
+  {% for link in nav.links %}
+  <li>
+    {% if link.links %}
+    <button class="usa-accordion-button usa-nav-link" aria-expanded="false" aria-controls="{{ link.id }}">
+      <span>{{ link.text }}</span>
+    </button>
+    <ul id="{{ link.id }}" class="usa-nav-submenu{% if nav_mega %} usa-megamenu usa-grid-full{% endif %}">
```

Comment:
So another note that best not addressed now:

If you need to use the mega menu here, you have to add both `usa-megamenu` and `usa-grid-full`. I wonder if we could use sass to mixin `usa-grid-full` into the megamenu class so only that's needed. Might improve the dev experience to only require one extra class.